mappings/community/convictions/DX12340A-convictions.json

```diff
@@ -476,18 +476,6 @@
             "lengthUnits": "Months",
             "terminationDate": "2018-01-23",
             "terminationReason": "ICMS Miscellaneous Event",
-            "currentOrderHeaderDetail" : {
-              "custodialTypeCode" : "B",
-              "custodialTypeDescription": "ORA Adult Custody (inc PSS)",
-              "mainOffenceDescription": "Dangerous driving/Aid, abet cause or permit reckless driving - 80200",
-              "status": "Released - On Licence",
-              "disposalDate": "{{now offset='-6 months' format='yyyy-MM-dd'}}",
-              "actualReleaseDate": "{{now offset='-6 months' format='yyyy-MM-dd'}}",
-              "licenceExpiryDate": "{{now offset='6 months' format='yyyy-MM-dd'}}",
-              "pssEndDate": "{{now offset='12 months' format='yyyy-MM-dd'}}",
-              "length": 12,
-              "lengthUnit": "Months"
-            },
```

Comment:
Looking at CCS swagger, I noticed that `currentOrderHeaderDetail` doesn't return on the probation-record endpoint but does on the single conviction endpoint as part of the sentence, so I have removed from the mock data and updated the pact schema.